zh/scale-a-tidb-cluster.md

```diff
@@ -1,20 +1,20 @@
 ---
-title: Kubernetes 上的 TiDB 集群扩缩容
-summary: 介绍如何在 Kubernetes 中对 TiDB 集群扩缩容。
+title: 手动扩缩容 Kubernetes 上的 TiDB 集群
+summary: 了解如何在 Kubernetes 上对 TiDB 集群手动扩缩容。
 aliases: ['/docs-cn/tidb-in-kubernetes/dev/scale-a-tidb-cluster/']
 ---
 
-# Kubernetes 上的 TiDB 集群扩缩容
+# 手动扩缩容 Kubernetes 上的 TiDB 集群
 
-本文介绍 TiDB 在 Kubernetes 中如何进行水平扩缩容和垂直扩缩容。
+本文介绍如何对部署在 Kubernetes 上的 TiDB 集群进行手动水平扩缩容和垂直扩缩容。
 
 ## 水平扩缩容
 
-TiDB 水平扩缩容操作指的是通过增加或减少节点的数量，来达到集群扩缩容的目的。扩缩容 TiDB 集群时，会按照填入的 replicas 值，对 PD、TiKV、TiDB 进行顺序扩缩容操作。扩容操作按照节点编号由小到大增加节点，缩容操作按照节点编号由大到小删除节点。目前 TiDB 集群使用 TidbCluster Custom Resource (CR) 管理方式。
+TiDB 水平扩缩容操作指的是通过增加或减少节点的数量，来达到集群扩缩容的目的。扩缩容 TiDB 集群时，会按照填入的 replicas 值，对 PD、TiKV、TiDB 按顺序进行扩缩容操作。扩容操作按照节点编号由小到大增加节点，缩容操作按照节点编号由大到小删除节点。目前 TiDB 集群使用 TidbCluster Custom Resource (CR) 管理方式。
 
 ### 扩缩容 PD、TiDB、TiKV
```

Comment:
@KanShiori Would you please help turn this section into clear steps (1, 2, 3...)? You may need to add more commands so users can copy directly. Thanks!